### ISSUE TITLE
getting error from isosurface plot

### ISSUE THREAD

**Issue by wilsonwcyiu** [DIPlib/diplib#75]
When I try to generate the isosurface image from "Actions -> isosurface plot...". I got the error message as shown from the image below.
Am I missing something?

![image](https://user-images.githubusercontent.com/13941660/122098146-be27a580-ce10-11eb-80d9-f1a0b7a6d49b.png)

**Comment by crisluengo** [comment]
You've found a bug that was fixed in October 2019. I assume you are using the old binary distribution we have in GitHub, which was released just a week or so before we found and fixed this bug.

Unfortunately we don't have a way to automatically build binary releases for DIPimage (the MATLAB dependency makes it hard to do this on public build services typically used for open-source software). The best solution is to follow [the build instructions for Windows here](https://github.com/DIPlib/diplib/blob/master/INSTALL_Windows.md), it requires you download some free build software, but otherwise is not hard at all.

The quick solution is just downloading the fixed function [here](https://github.com/DIPlib/diplib/blob/master/dipimage/dipisosurface.m) and overwrite the one in your installation. This does not, however, fix all the other issues nor add all the other improvements of the software we've made since the release you're using.
